packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1209,6 +1209,18 @@ export default class Meeting extends StatelessWebexPlugin {
     };
   }
 
+  /**
+   * Temporary func to return webex object,
+   * in order to access internal plugin metrics
+   * in the utils file.
+   * @internal
+   * @returns {object} webex object
+   */
+  getWebexObject() {
```

Comment:
I wasn't entirely sure it's safe to use `meeting.webex` from outside the meeting class?

I think I mentioned it's temporary because I do hope one day we can have the singleton approach back.. I can remove the "temporary" thing tho.

Reply:
dont know if you want to do this now, but you can move the catch for join into the  parent function then you don't need getWebexObject function

typically util files just do data processing and gives back the results for parent to make request or send data